tool.go

```diff
@@ -115,10 +115,8 @@ func main() {
 							s.Watcher.Add(name)
 						}
 					}
-					if err := s.BuildFiles(args, pkgObj, currentDirectory); err != nil {
-						return err
-					}
-					return nil
+					err := s.BuildFiles(args, pkgObj, currentDirectory)
+					return err
```

Comment:
If you're going to make the change, why not go all the way to:

    return s.BuildFiles(args, pkgObj, currentDirectory)

Reply:
Imo it's more readable to see that it returns an error, but it isn't that strong opinion :)

Reply:
That would work too, but personally, I don't mind (in fact, I slightly prefer) it as written. It has to do with where I perceive the emphasis to be when reading the code.

When I see:

```Go
err := s.BuildFiles(args, pkgObj, currentDirectory)
return err
```

I read that as "do `s.BuildFiles`, then return the error result."

When I see:

```Go
return s.BuildFiles(args, pkgObj, currentDirectory)
```

I read that as "return the result of building files", which seems to put more emphasis on the return value, but in fact the return value is pretty insignificant (it doesn't return a package it built, just a nil error most of the time).

Anyway, this is a personal style and both ways are fine.